script/Ciel/Day 3/03_01_CIEL03_1.txt

```diff
@@ -11,7 +11,7 @@ I wasn't used to the route, so I must have paced myself wrong.
 [sha:6d00dc71bcca4185f381b831e68bdb3c4e384a44]{
 -- Page 1, Offset 4854.
 -- 朝の雑談時間も終わりにさしかかり、生徒の大部分は自分の机に座っている。
-The morning chatter is starting to dwindle, and most of the students have gone back to their desks.
+The morning chatter is starting to dwindle, and most of the students are returning to their desks.
```

Comment:
prefer original here imo

Reply:
Gotta disagree. "return to your seats" is a set phrase in a school environment.